credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -198,6 +221,19 @@ def _remove_initial_junk(self, snippet):
         """ Remove junk from the beginning of a snippet.
         """
         return re.sub(
-            r"^((\s*|\ *)\@\@.*\@\@(\s*|\ *)|(\s*|\ *)\+(\s*|\ *)|(\s*|\ *)\-(\s*|\ *)|(\s*|\ *))",
+            r'^((\s*|\ *)\@\@.*\@\@(\s*|\ *)|(\s*|\ *)\+(\s*|\ *)|(\s*|\ *)\-(\s*|\ *)|(\s*|\ *))',
             "",
             snippet).strip()
+
+    def _check_private_key(self, snippet):
+        """Check if this snippet is a private key
+
+        Args:
+            snippet (str): A code snippet containing a secret
+
+        Returns:
+            boolean: True if this is a header of a private key, False otherwise.
+        """
```

Comment:
The style of this doctoring is not consistent with the rest on the codebase (more on this in the final comment)